test/hotspot/jtreg/compiler/jvmci/TestEnableJVMCIProduct.java

```diff
@@ -103,7 +112,11 @@ static void test(String explicitFlag, Expectation... expectations) throws Except
                     output.stdoutShouldMatch("No JVMCI compiler found");
                 }
             } else if (flag.equals("-XX:+UseGraalJIT")) {
-                output.shouldContain("jvmci.Compiler=graal");
```

Comment:
Does changing shouldContain() to stdoutShouldContain() also solve the problem?

Reply:
I don't think so as the output is sent to HotSpot's log stream (i.e. `tty`) which I believe goes stdout by default.